NewForce-at-Mountwest/bangazon-workforce-management-the-final-slappers
# Created initial project files

# Description
Setup initial VS project, added Model files, added a Nav Bar to Shared/_Layout.cshtml

## Steps to Test
- Ensure that the VS project can be opened
- Ensure the VS project contains Model files for the tables presented in the BangazonWorkforce database
- Press CTRL + F5 and make sure the Nav Bar appears on the Razor site

## Checklist
- [ ] The above steps all produce the desired results
